### ISSUE TITLE
Unable to install jupyterlab extension

### ISSUE THREAD

**Issue by nik123** [jupyter/nbdime#658]
I cannot install jupyterlab extension for nbdime. Here is what I do:

```
pip install nbdime
nbdime extensions --enable
```

I get the following error:
```
Couldn't find package "nbdime@^6.1.2" required by "nbdime-jupyterlab@file:../extensions/nbdime-jupyterlab-2.1.1.tgz" on  the "npm" registry.
info Visit https://yarnpkg.com/en/docs/cli/install for documentation about this command.

npm dependencies failed to install
```

My environment is:
* jupyterlab==3.5.0
* Python 3.7.13

**Comment by nik123** [comment]
It seems I was able to finally install a nbdime. My machine is in internal network and all the outside packages are available through the https proxy with self-signed certificates. To install nbdime via a proxy I had to to the following steps:
1. Updated Jupyterlab settings in my virtual environment by updating `lib/python3.7/site-packages/jupyterlab/staging/yarn.js`: I found there a linewith `strict-ssl: true` and replaced it with `strict-ssl: false`.
2. Run  `REQUESTS_CA_BUNDLE=/path/to/file SSL_CERT_FILE=/path/to/file nbdime extensions --enable`

I'm fine with passing env variables to `nbdime extensions`, but is there a way to avoid editing `yarn.js` file in `site-packages`?
